openebs/openebs-provisioner.go

```diff
@@ -43,6 +43,8 @@ const (
 	// BetaStorageClassAnnotation represents the beta/previous StorageClass annotation.
 	// It's currently still used and will be held for backwards compatibility
 	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+	//defaultFSType
+	defaultFSType = "ext4"
```

Comment:
can we have a map of supported fsTypes so that we can validate if the requested type is valid or not?

Reply:
Yes, we can have the map of supported fstype to validate them.

Reply:
done.